packages/rocketchat-webrtc/WebRTCClass.coffee

```diff
@@ -803,9 +803,11 @@ WebRTC = new class
 
 
 Meteor.startup ->
-	RocketChat.Notifications.onUser 'webrtc', (type, data) =>
-		if not data.room? then return
+	Tracker.autorun ->
```

Comment:
Can you stop the autorun? `Tracker.autorun (c) ->` then you call `c.stop()` after your if.